src/Sdk/NewsroomSubscriptions/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Contact, UploadcareImage } from '../../types';
+
+export interface NewsroomSubscriptionCreateRequest {
+    email_address: string;
+    locale?: string;
+    url?: string | null;
+    ip_address?: string | null;
+    visitor_uid?: string | null;
+    session_uid?: string | null;
+    contact?: {
+        contact_type: Contact['contact_type'];
+        company_name: Contact['company_name'];
+        first_name?: Contact['first_name'];
+        last_name?: Contact['last_name'];
+        gender?: Contact['gender'];
+        function_name?: Contact['function_name'];
+        organisations?: string[];
```

Comment:
`first_name`, `last_name`, `gender`, `function_name` and `organisations` is only available when `contact_type === "person". I'd mention this in properties comments.

Reply:
Nice suggestion, I added comments. Thanks!